src/js/base/ripe.js

```diff
@@ -1195,7 +1195,15 @@ ripe.Ripe.prototype.bindImage = function(element, options = {}) {
  */
 ripe.Ripe.prototype.bindConfigurator = function(element, options = {}) {
     options = Object.assign({}, { format: this.format }, options);
-    const config = new ripe.Configurator(this, element, options);
+    let config = null;
+    switch (options.type) {
+        case "csr":
+            config = this.bindConfiguratorCsr(element, options);
+            break;
+        case "prc":
+        default:
+            config = this.bindConfiguratorPrc(element, options);
+    }

```

Comment:
SWhouldn't this return instead of assigning the config? I'm a little confused after seeing what `bindConfiguratorPrc` does. Also `bindConfiguratorCsr` doesn't exist

Reply:
`bindConfiguratorCsr ` is defined within the external CSR package, in ripe-tech/sandbox. There was the decision to isolate CSR to outside the SDK, and it generates this situation, unfortunately

Reply:
But about my other question, why there is the `config = ` part? Checking what the functions do, it already does the `BindInteractable(config);`

Reply:
That's actually a great point! I'm going to try and remove that config, and just return the result of `bindConfiguratorPrc`, see if it works. I might be binding the interactable twice. I'll let you know!

Reply:
![image](https://user-images.githubusercontent.com/22790704/143270173-ba058ae6-a872-43e7-bb11-98b5e06d459f.png)

Seems like everything is still working as expected! Changing.